spec/acceptance/init_spec.rb

```diff
@@ -5,57 +5,41 @@
   include Beaker::TaskHelper::Inventory
   include BoltSpec::Run
 
-  def module_path
-    RSpec.configuration.module_path
-  end
-
-  def config
-    { 'modulepath' => module_path }
-  end
-
-  def inventory
-    hosts_to_inventory.merge('features' => ['puppet-agent'])
-  end
-
-  def run(params)
-    run_task('service', 'default', params, config: config, inventory: inventory)
-  end
-
-  osfamily_fact = fact('osfamily')
+  osfamily_fact = os[:family]
 
   package_to_use = ''
   before(:all) do
     if osfamily_fact != 'windows'
-      if osfamily_fact == 'RedHat' && fact('operatingsystemrelease') < '6'
-        run('action' => 'stop', 'name' => 'syslog')
+      if osfamily_fact == 'redhat' && os[:release].to_i < 6
+        task_run('service', 'action' => 'stop', 'name' => 'syslog')
       end
       package_to_use = 'rsyslog'
       apply_manifest_on(default, "package { \"#{package_to_use}\": ensure => present, }")
```

Comment:
You can use https://github.com/puppetlabs/bolt/blob/master/lib/bolt_spec/run.rb#L50 for this and not have to depend on using beaker.